staging/src/k8s.io/client-go/plugin/pkg/client/auth/exec/metrics.go

```diff
@@ -58,3 +71,23 @@ func (c *certificateExpirationTracker) set(a *Authenticator, t time.Time) {
 		c.metricSet(&earliest)
 	}
 }
+
+// incrementCallsMetric increments a global metrics counter for the number of calls to an exec
+// plugin, partitioned by exit code. The provided err should be the return value from
+// exec.Cmd.Run().
+func incrementCallsMetric(err error) {
```

Comment:
suggest using https://golang.org/pkg/errors/#As for these two error types to be able to handle wrapped errors

Reply:
18b42aeab316651173d904b57827099712e6493f